packages/mostly-equal/src/mostly-equal-chai-plugin.ts

```diff
@@ -1,16 +1,47 @@
 /// <reference types="chai" />
 
+import { Replacer } from './types';
 import { checkExpectValues, errorString } from './mostly-equal';
 import { safePrint } from './safe-print';
 
 export interface MostlyEqualOptions {
     maxDepth?: number;
+    replacers?: Replacer[];
 }
 
+const globalOptions: MostlyEqualOptions = {
+    maxDepth: 10,
+    replacers: [],
+};
+
+export const setGlobalOptions = (maxDepth: number, replacers?: Replacer[]) => {
+    globalOptions.maxDepth = maxDepth;
+    globalOptions.replacers = replacers;
+};
+
+export const tempSetGlobalOptions = (
```

Comment:
I hate this API, but if you're going to keep it, please rename to something helpful
(setSuiteOptions or something)

Reply:
done